src/test/resources/META-INF/persistence.xml

```diff
@@ -156,6 +156,8 @@
 		<!-- Profile package. -->
 		<class>org.sigmah.server.domain.profile.GlobalPermission</class>
 		<class>org.sigmah.server.domain.profile.OrgUnitProfile</class>
+		<class>org.sigmah.server.domain.profile.MainOrgUnitProfile</class>
+		<class>org.sigmah.server.domain.profile.SecondaryOrgUnitProfile</class>
```

Comment:
Also, it seems that 5 classes should be declared in this file while they are not (`org.sigmah.server.domain.Framework*`)